compressed_image_transport/src/compressed_publisher.cpp

```diff
@@ -120,6 +120,43 @@ void CompressedPublisher::advertiseImpl(
     RCLCPP_DEBUG(logger_, "%s was previously declared", jpeg_quality_param_name.c_str());
     config_.jpeg_quality = node->get_parameter(jpeg_quality_param_name).get_value<int64_t>();
   }
+
+  std::string tiff_res_unit_param_name = param_base_name + ".tiff.res_unit";
+  rcl_interfaces::msg::ParameterDescriptor tiff_res_unit_description;
+  tiff_res_unit_description.description = "tiff ResUnit";
```

Comment:
I suggest being a bit more descriptive.
```suggestion
  tiff_res_unit_description.description = "tiff resolution unit";
```

Reply:
Done in https://github.com/ros-perception/image_transport_plugins/pull/75/commits/403c4b20a1befd9f9d0b4b41ff68658f9ac57558